include/wil/stl.h

```diff
@@ -187,6 +187,10 @@ namespace wil
 
     using zstring_view = basic_zstring_view<char>;
     using zwstring_view = basic_zstring_view<wchar_t>;
+

```

Comment:
I can't recall if we ever added a section to the documentation for zwstring_view.  If we did, would you be willing to add a note about the new literal support?  That is definitely helpful to have and documentation is how people will learn it exists.